docs/_docs/development/setup.md

```diff
@@ -113,6 +113,22 @@ FREEGENES_TWIST_EUTOKEN=xxxxxxxxxxxxxxxxxxxxxx
 If you don't define these variables, they will be set to None and the
 option to import orders from Twist simply won't be presented.
 
+### Map Tokens
+
+The Orders Map (a map of all historical orders found at the route /map/orders) is generated using Mapbox GL JS (https://docs.mapbox.com/mapbox-gl-js/api/).
+
+In order for the JS library to generate the map, you need to supply a valid Mapbox access token ("mapboxgl.accessToken").
+
```

Comment:
I would say `mapboxgl.accessToken` in the JavaScript that creates the map so it's clear - we have lots of different references for variable, and it should be clear where / what language you are referring to.

Reply:
I'm a bit confused. Can you explain what you mean?

Reply:
Write this:

> In order for the JS library to generate the map, you need to supply a valid Mapbox access token ("mapboxgl.accessToken" in the JavaScript that creates the map). This variable is populated from the `MAPBOX_ACCESS_TOKEN` in your `settings/config.py`:

Reply:
This is done.